src/httk/iface/vasp_if.py

```diff
@@ -430,12 +438,421 @@ def set_final(results, match):
         def read_energy(results, match):
             self.final_energy_with_entropy = match.group(1)
             self.final_energy = match.group(2)
+
+        def read_stress_tensor(results, match):
+            # NOTE: Stress tensor values are in a different order in OUTCAR
+            # compared to Voigt notation. In terms of Voigt notation OUTCAR
+            # stress tensor is ordered as:
+            # 1 2 3 6 4 5
+            # The unit is kB = 0.1 GPa.
+            # Also, OUTCAR gives the stress tensor with a minus sign.
+            self.stress_tensor = [match.group(1), match.group(2),
+                                  match.group(3), match.group(4),
+                                  match.group(5), match.group(6)]
+            # Due to these VASP conventions, provide also the stress tensor
+            # in the normal Voigt order and in units of GPa:
+            self.stress_tensor_voigt_gpa = [
+                str(np.round(-float(self.stress_tensor[0])/10, 5)),
+                str(np.round(-float(self.stress_tensor[1])/10, 5)),
+                str(np.round(-float(self.stress_tensor[2])/10, 5)),
+                str(np.round(-float(self.stress_tensor[4])/10, 5)),
+                str(np.round(-float(self.stress_tensor[5])/10, 5)),
+                str(np.round(-float(self.stress_tensor[3])/10, 5)),
+                ]
+
         results = micro_pyawk(self.ioa, [
-                              ["^ *energy *without *entropy= *([^ ]+) *energy\(sigma->0\) *= *([^ ]+) *$", None, read_energy],
+                              ["^ *energy *without *entropy= *([^ ]+) "+
+                               "*energy\(sigma->0\) *= *([^ ]+) *$", None, read_energy],
                               ["FREE ENERGIE", None, set_final],
+                              # ["^ *in kB" + " *([^ \n]+)"*6, None, read_stress_tensor]
                               ], results, debug=False)
         self.parsed = True
```

Comment:
I cannot see how the function "read_stress_tensor" here will be called, given the commented out argument to micro_pyawk. Is it kept here to be commented back in, if desired? (In that case we probably rather want a flag argument.)